content/stories/fungus-group.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Community Fungus Study
+location: Norfolk,
+images:
```

Comment:
I think you need one more level in here for this to be a list of images - but also, just re-name it as image and raise a new ticket: Allow Multiple images in a story.

Reply:
Hmm I think it is a list? It's a list of records that look like `{src: String, alt: String}` rather than `{image: {src: String, alt: String}}` which didn't seem necessary. It's just that there's only one image in this example list.

Reply:
You are right. Sorry! I thought we could get away with less complication here.